.github/release-drafter.yml

```diff
@@ -0,0 +1,29 @@
+name-template: 'v$RESOLVED_VERSION 🌈'
+tag-template: 'v$RESOLVED_VERSION'
+categories:
+  - title: '🚀 Features'
+    labels:
+      - 'feature'
+      - 'enhancement'
+  - title: '🐛 Bug Fixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
+      - 'bug'
+change-template: '- $TITLE @$AUTHOR (#$NUMBER)'
+change-title-escapes: '\<*_&' # You can add # and @ to disable mentions, and add ` to disable code blocks.
```

Comment:
What does this do... :thinking:

Reply:
Characters to escape in $TITLE when inserting into change-template so that they are not interpreted as Markdown format characters.

Reply:
Ah I see! I do backticks sometimes, but I want that as markdown formatting